lua/telekasten.lua

```diff
@@ -1998,15 +1998,9 @@ local function FollowLink(opts)
     end
 
     if kind == "tag" then
-        if atcol ~= nil then
-            tag = taglinks.get_tag_at(
-                vim.api.nvim_get_current_line(),
-                atcol,
-                M.Cfg
-            )
-        end
         search_mode = "tag"
-        title = tag
+	      vim.cmd("normal yiw")
+        title = vim.fn.getreg('"0')
```

Comment:
As fixed in identified in #184 and fixed in #259, we should ont override the default register. 
As in #259, it would be best to first save then reset the register content.